autograder/plagiarism_detection/lexers/Python3Lexer.py

```diff
@@ -2,6 +2,7 @@
 from antlr4 import *
```

Comment:
It seems to me like you don't have to edit the library but could use an adapter instead. Please, think about such a solution because leaving a library in the project is still not the best idea.